sparsecoding/data/datasets/datasets.py

```diff
@@ -90,46 +87,45 @@ def whiten(data):
     wdata - scalar (N,n)
         whitened data
     """
-    
+
     cdata = data.T-data.mean()
-    
+
     cov = np.cov(cdata)
     w, v = np.linalg.eig(cov)
     diagw = np.diag(np.real(1/(w**0.5)))
-    
+
     wdata = v@diagw@v.T@cdata
     return wdata.T.astype(np.float32)
 
 
-def whiten_dataset(dataset,plot=False,title='',n_comp=-1):
+def whiten_dataset(dataset, plot=False, title='', n_comp=-1):
     from torchsmodel import sparsecoding
```

Comment:
Remove import

Reply:
This isn't in the repo. I think it's code from your previous implementation @belsten. It's called later in the code too. How is it passing tests? Are you perhaps not calling this function?